lua/resession/init.lua

```diff
@@ -174,7 +174,7 @@ M.delete = function(name, opts)
   local files = require("resession.files")
   local util = require("resession.util")
   if not name then
-    local sessions = M.list()
+    local sessions = M.list({ opts.dir })
```

Comment:
For this to work, you'll also have to pass the opts in on line 187, right?

Reply:
Right, thanks for catching that!

Reply:
Oh, also I didn't notice this on the first review, but this should be `{ dir = opts.dir }`

Reply:
Haha, my bad! updated that as well.